content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -224,29 +137,28 @@ export KUBECONFIG=/etc/kubernetes/admin.conf
 ```
 
 Make a record of the `kubeadm join` command that `kubeadm init` outputs. You
-will need this in a moment.
+need this command to [join nodes to your cluster](#join-nodes).
 
 The token is used for mutual authentication between the master and the joining
-nodes.  The token included here is secret, keep it safe as anyone with this
+nodes.  The token included here is secret. Keep it safe, because anyone with this
 token can add authenticated nodes to your cluster. These tokens can be listed,
-created and deleted with the `kubeadm token` command. See the
-[reference guide](/docs/reference/setup-tools/kubeadm/kubeadm-token/) for more details.
+created, and deleted with the `kubeadm token` command. See the
+[kubeadm reference guide](/docs/reference/setup-tools/kubeadm/kubeadm-token/).
```

Comment:
OK, so you don't actually need to keep a separate record of them, since you can run those commands at any time.